### PR TITLE
Feature gridsection sort

### PR DESCRIPTION
**Pull request by kctani**

### CODE REVIEW

**Comment by aparcar** [comment]
Hi, this sounds like a great feature! Please clean up those commits, we usually don't to *squash* commits here but try to keep the log clean. Also I'd be curious to see a screenshot of this.

File: modules/luci-mod-cname/Makefile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#
+# Copyright (C) 2008-2014 The LuCI Team <luci@lists.subsignal.org>
```
Comment (aparcar) [inline modules/luci-mod-cname/Makefile:2]: Please don't add a copyright like this, leave it or add your own in SPDX format

**Comment by systemcrash** [comment]
@kctani Can you address the conflicts for this PR so it's ready for merge? Also, if you manage, attach an mp4 screen cap demonstrating it in use.

**Comment by kctani** [comment]
I gave up on this pull request two years ago. I thought I had canceled it. In any case to the best of my knowledge sort is now implemented.

**Comment by systemcrash** [comment]
OK, thanks :)

**Pull request closed without merging**